DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+(function () {
```

Comment:
Include the standard header and copyright notice

Reply:
Done